salsa_webqa/library/control_test.py

```diff
@@ -69,6 +69,8 @@ def gid(self, searched_id):
          If local execution parameter is "True", function will try to search for parameter in local configuration file.
          If such parameter is not found or there is an error while reading the file, server (default) configuration
          file will be used instead. """
+        if self.configs is None:
```

Comment:
I can add support to have config files, then I guess I need to use some strict names to find if project contain config for non selenium tests, like "non_selenium_config.properties" ?